plugins/db_manager.py

```diff
@@ -174,6 +175,62 @@ async def acl_show(ctx: Context, acl: str) -> None:
 
     await ctx.send(format("{!b:yaml}", yaml.dump(data)))
 
+@acl_command.command("find")
+@privileged
+async def acl_find(ctx: Context, type: str, id: int) -> None:
+    """ Find all ACLs that directly reference the given user, role, channel, or category ID"""
+
+    if ctx.guild is None:
+        raise UserError("Can only be used in a guild")
+
+    if type not in ["role", "channel", "category", "user"]:
+        raise UserError("Invalid type, must be one of `user`, `role`, `channel`, or `category`")
+    if type == "role" and ctx.guild.get_role(id) is None:
+        raise UserError(format("No role with ID: {!i}", id))
+    if type in ("channel", "category") and ctx.guild.get_channel(id) is None:
+        raise UserError(format("No channel with ID: {!i}", id))
+    if type == "user" and ctx.guild.get_member(id) is None:
+        raise UserError(format("No user with ID: {!i}", id))
```

Comment:
Don't see why these checks have to be carried out at all. I appreciate the ergonomics of specifying a user and have the bot automatically look up their roles, but I don't think it should prevent you from doing a lookup on e.g. a deleted role.